DataManagementSystem/Client/FTSJob.py

```diff
@@ -35,14 +35,14 @@
 from DIRAC.Resources.Catalog.FileCatalog     import FileCatalog
 from DIRAC.Core.Utilities.ReturnValues import returnSingleResult
 
-import fts3.rest.client.easy as fts3
+import fts3.rest.client.easy as fts3 #pylint: disable=import-error
```

Comment:
of course there is :-)
pip install "git+https://gitlab.cern.ch/fts/fts-rest.git@master"
Just replace master with the version you want